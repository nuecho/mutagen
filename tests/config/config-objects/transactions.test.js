cfgObjectTests("transactions", "config-objects/transactions-config.json");
