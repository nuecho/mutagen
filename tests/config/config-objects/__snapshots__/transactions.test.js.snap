// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen transactions config import-export] should create the transactions: the transactions has been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ transaction => name: 'cfgObjectTest', type: 'callData', tenant: 'Environment'
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen transactions config import-export] should update the transactions: the transactions has been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ transaction => name: 'cfgObjectTest', type: 'callData', tenant: 'Environment'
Completed. 1 object(s) imported.
",
}
`;
